README.md

```diff
@@ -125,8 +125,23 @@ Region of interest (ROI) should be marked with `torch.aten_profiler_start()` and
 ```python
 import torch
 
-torch.aten_profiler_start()
+# start of ROI
+torch.aten_profiler.enable()
 x = torch.randn(10)
 y = x + x
-torch.aten_profiler_end()
+# end of ROI
+torch.aten_profiler.disable()
+
+# print out top level kernel table
+torch.aten_profiler.fancy_print()
```

Comment:
Printing to a file might be nice! And to a default file if filename is not given...?

Reply:
Added a `fancy_print_to_file()` call! It prints to `profiling.txt` by default. ;)